runtime.ts

```diff
@@ -314,6 +317,8 @@ class TypeScriptHost implements ts.LanguageServiceHost {
         resolvedFileName = resolveModuleName("deno.d.ts", "/$asset$/");
       } else if (name === "typescript") {
         resolvedFileName = resolveModuleName("typescript.d.ts", "/$asset$/");
+      } else if (name === "http") {
+        resolvedFileName = resolveModuleName("http.d.ts", "/$asset$/");
```

Comment:
Can you merge all the http stuff into the Deno namespace? Maybe it can be split up later, but for now I prefer to keep it flat.

Reply:
Sure